### PR TITLE
minor spelling edits and clarifications for the dev tutorial

### PR DESCRIPTION
**Pull request by mbmcgarry**

### CODE REVIEW

File: source/arche/tutorial/setup.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ Make a new input file that is a copy of the test input file
```
Comment (vcloitre) [inline source/arche/tutorial/setup.rst:62]: All sed -i'' "s/... commands didn't work for me (lines 32, 33, 34 and 70). I had to add a space between the -i and the quotation marks to make it work.
For example, for line 32, I wrote
$ sed -i '' "s/tutorial_facility/storage/g" src/storage*

Also a typo line 6 : "the the Cyclus"->"the Cyclus"

File: source/arche/tutorial/testing.rst [hunk 2]
```diff
@@ -73,18 +73,11 @@ You can then test the test's initial condition
...
 .. code-block:: c++
 
-    TEST_F(StorageTest, Tock) {
-      double qty = 42;
-      facility->input.Push(cyclus::NewBlankMaterial(qty));
-      EXPECT_DOUBLE_EQ(facility->input.quantity(), qty);
-      EXPECT_DOUBLE_EQ(facility->inventory.quantity(), 0);
       EXPECT_NO_THROW(facility->Tock());
-      // Test Storage specific behaviors of the Tock function here
-    }
```
Comment (vcloitre) [inline source/arche/tutorial/testing.rst:81]: I had to replace all "facility" with "src_facility_" in the test for the Tock function or else the build failed. Was it working for  you with "facility" @mbmcgarry ?

Comment (gidden) [inline source/arche/tutorial/testing.rst:81]: That is a documentation update which has not reached the stubs yet. FYI.

On Mon, May 18, 2015 at 2:10 PM, vcloitre notifications@github.com wrote:

> In source/arche/tutorial/testing.rst
> https://github.com/gidden/cyclus.github.com/pull/3#discussion_r30538076:
> 
> > ```
> >    EXPECT_NO_THROW(facility->Tock());
> > ```
> > -      // Test Storage specific behaviors of the Tock function here
> > -    }
> 
> I had to replace all "facility" with "src_facility_" in the test for the
> Tock function or else the build failed. Was it working for you with
> "facility" @mbmcgarry https://github.com/mbmcgarry ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gidden/cyclus.github.com/pull/3/files#r30538076.

## 

Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

**Comment by gidden** [comment]
I merged the commit into my rebased `devtut` branch

**Pull request closed without merging**